src/components/ContentItems.js

```diff
@@ -10,13 +10,13 @@ class ContentItems extends Component {
 
       contentItems.map((content) =>
         <div className="coa-content_items" key={content.id}>
-          {(content.type === 'application_block') && (
+          {/*(content.type === 'application_block') && (
             <div className="coa-section">
               <h4>{content.value.description}</h4>
               INSERT {content.type} app HERE
               { content.value.url && <div dangerouslySetInnerHTML={{__html: content.value.url}} /> }
             </div>
-          )}
+          )*/}
```

Comment:
ewww. Is it ok if we delete this code instead of commenting it out? Or was this just a quick fix for usability testing? Maybe we add a `{/* TODO */}` or something easy to Command-F?

Reply:
Yea, this was removed for usertesting. I can update with a comment.